docs/reference/deprecation-policy.md

```diff
@@ -70,97 +70,193 @@ API version at least as stable is released.**
 GA API versions can replace GA API versions as well as beta and alpha API
 versions.  Beta API versions *may not* replace GA API versions.
 
-**Rule #4: Other than the most recent API versions in each track, older API
+**Rule #4a: Other than the most recent API versions in each track, older API
 versions must be supported after their announced deprecation for a duration of
 no less than:**
 
    * **GA: 1 year or 2 releases (whichever is longer)**
-   * **Beta: 3 months or 1 release (whichever is longer)**
+   * **Beta: 6 months or 2 releases (whichever is longer)**
    * **Alpha: 0 releases**
 
-This is best illustrated by example.  Imagine a Kubernetes release, version X,
-which supports a particular API group.  A new Kubernetes release is made every
-approximately 3 months (4 per year).  The following table describes which API
-versions are supported in a series of subsequent releases.
+**Rule #4b: The "default" API version for a given group may not advance util
```

Comment:
@lavalamp Is "preferred" the standard terminology for this? How do we talk about it in the discovery API?

Reply:
Yes, "preferred". The api-preferred and storage-default versions don't need to be the same. I am still thinking about whether this rule 4b is actually necessary.

Reply:
Given our kubectl version-skew problems even with our weak guarantee of 1 release, I suspect it is.

For the storage version, it definitely is.